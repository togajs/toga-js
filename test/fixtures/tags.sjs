/** description text */

/**
 * description
 * @module {Type.<Type,Type(Type,Type.<Type>)>} [name={}] - description text
 * @module {Type} name description text
 * @module [name] - description text
 * @module name - description text
 * @module name
 */

/** @extends {Type.<Type,Type(Type,Type.<Type>)>} [name={}] - description text */
/** @extends {Type} name description text */
/** @extends [name] - description text */
/** @extends name - description text */
/** @extends name */

/** @tag {Type.<Type,Type(Type,Type.<Type>)>} - description text */
/** @tag {Type} description text */
/** @tag - description text */
/** @tag description text */
/** @tag */

/*! ignore */
/*!
 * ignore
 */

/* ignore */
/*
 * ignore
 */
