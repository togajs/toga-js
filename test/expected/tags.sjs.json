{
  "type": "Document",
  "blocks": [
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "description text",
      "tags": []
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "description\n",
      "tags": [
        {
          "tag": "module",
          "type": "{Type.<Type,Type(Type,Type.<Type>)>}",
          "name": "[name={}]",
          "description": "description text\n"
        },
        {
          "tag": "module",
          "type": "{Type}",
          "name": "name",
          "description": "description text\n"
        },
        {
          "tag": "module",
          "name": "[name]",
          "description": "description text\n"
        },
        {
          "tag": "module",
          "name": "name",
          "description": "description text\n"
        },
        {
          "tag": "module",
          "name": "name"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "extends",
          "type": "{Type.<Type,Type(Type,Type.<Type>)>}",
          "name": "[name={}]",
          "description": "description text"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "extends",
          "type": "{Type}",
          "name": "name",
          "description": "description text"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "extends",
          "name": "[name]",
          "description": "description text"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "extends",
          "name": "name",
          "description": "description text"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "extends",
          "name": "name"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "tag",
          "type": "{Type.<Type,Type(Type,Type.<Type>)>}",
          "description": "description text"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "tag",
          "type": "{Type}",
          "description": "description text"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "tag",
          "description": "description text"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "tag",
          "description": "description text"
        }
      ]
    },
    {
      "type": "Code",
      "contents": ""
    },
    {
      "type": "Comment",
      "description": "",
      "tags": [
        {
          "tag": "tag"
        }
      ]
    },
    {
      "type": "Code",
      "contents": "/*! ignore */\n/*!\n * ignore\n */\n\n/* ignore */\n/*\n * ignore\n */"
    }
  ],
  "nav": {}
}
